src/Lykke.Job.TransactionHandler/Queues/Models/ClientTradesFactory.cs

```diff
@@ -18,60 +18,42 @@ public ClientTradesFactory(IAssetsServiceWithCache assetsServiceWithCache)
             _assetsServiceWithCache = assetsServiceWithCache;
         }
 
-        public async Task<ClientTrade[]> Create(string orderId, string clientId, string assetPairId, TradeQueueItem.TradeInfo trade, double marketVolume, double limitVolume, double price)
+        public async Task<ClientTrade[]> Create(TradeQueueItem.MarketOrder order, List<TradeQueueItem.TradeInfo> trades, string clientId)
```

Comment:
This method looks very similar to https://github.com/LykkeCity/Lykke.Job.TransactionHandler/pull/112/files#diff-35e5dc76242c217b9741965de6b776f8R15
Can it be reused?

Reply:
yes it can be refactored in some way, but this task is about Hotfix, and should be done asap. Anyway, soon TX handler will be refactored as I know.